### ISSUE TITLE
VESA Monitor

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/poang-hack#2]
Requirements:

+ [ ] VESA mount
+ [ ] Slim Besel
+ [ ] High Resolution (min QHD)
+ [ ] Low power consumption

https://www.pccomponentes.pt/viewsonic-vx-series-vx3276-2k-mhd-315-led-ips-quadhd
![image](https://user-images.githubusercontent.com/194400/89538638-bbbcee00-d7f2-11ea-9f07-a8e2582be370.png)

**Comment by nelsonic** [comment]
Lenovo on special:  https://www.pccomponentes.pt/lenovo-c32q-20-315-led-quadhd-freesync
![image](https://user-images.githubusercontent.com/194400/94004356-7a79a100-fd94-11ea-9f76-dbca269c62a5.png)

Think I still prefer the ViewSonic as it has thinner bezels and a metal construction. 💭 
Definitely prefer the _Black_ frame though ... 🙄

**Comment by nelsonic** [comment]
LG 35WN75C-B VA 35" UWQHD 21:9 100Hz with 100mm VESA
https://www.pcdiga.com/monitor-curvo-lg-35wn75c-b-va-35-uwqhd-21-9-100hz-freesync
![image](https://user-images.githubusercontent.com/194400/100020500-d5049d00-2dd7-11eb-8c28-d9767fe4b3b1.png)

This monitor is more than _double_ the price of others but I think it might be _worth_ it. 💭 
For context the _reason_ I'm considering getting a _much_ bigger (ultra-wide) monitor is because the Screen of the MacBook Pro I've been using for work has died: https://github.com/nelsonic/nelsonic.github.io/issues/815#issuecomment-732497118

**Comment by nelsonic** [comment]
This is the next level up: LG 38WN75C-B IPS **38"** WQHD+ 21:9 75Hz
https://www.pcdiga.com/monitor-curvo-lg-38wn75c-b-ips-38-wqhd-21-9-75hz
![image](https://user-images.githubusercontent.com/194400/100031944-c1186580-2dee-11eb-9050-e0b7a843d10a.png)

The 3840 x 1600 resolution means an extra 160 vertical pixels which might not sound like much, but it's _noticeable_.
see: https://www.performance-computer.com/reviews/lg-38bk95c-w/

[![image](https://user-images.githubusercontent.com/194400/100032420-e8236700-2def-11eb-899d-93e2a08d75f1.png)](https://youtu.be/jR_oP_9Pisw)

**Comment by nelsonic** [comment]
There isn't much point buying a monitor that can do more than **60Hz** refresh rate 
because Macbook Pro can only do 4096x2304 at 60 Hz see:
https://everymac.com/systems/apple/macbook/macbook-retina-faq/how-to-connect-monitor-to-macbook-retina-display.html

Apple appears to think this is "enough". Their $5k Pro XDR Display is also only **60Hz**! 💭

**Comment by nelsonic** [comment]
This is nice for a 34" ultrawide:
https://www.worten.pt/informatica-e-acessorios/monitores/monitores-pc/monitor-gaming-curvo-lenovo-g34w-10-34-1-ms-144-hz-freesync-7177604
![image](https://user-images.githubusercontent.com/194400/100033465-2c176b80-2df2-11eb-9326-1cbe4594c1ce.png)

But the high refresh rate will be completely wasted on me as I don't intend to buy a discrete graphics card. 💭

**Comment by nelsonic** [comment]
I really like the look of this SAMSUNG S32R750UEU 
https://www.worten.pt/informatica-e-acessorios/monitores/monitores-pc/monitor-samsung-s32r750ueu-32-4k-ultra-hd-va-6929606
![image](https://user-images.githubusercontent.com/194400/100033663-8c0e1200-2df2-11eb-9611-9a152d076acf.png)

https://displaysolutions.samsung.com/monitor/detail/1409/S32R750U
![image](https://user-images.githubusercontent.com/194400/100033840-dc856f80-2df2-11eb-831e-bebd06e58d88.png)

But ultimately, it can be replicated by just buying a nice desk mount and any other 32" 4K monitor. 💭

**Comment by nelsonic** [comment]
LG 32QK500-C: https://www.pccomponentes.pt/lg-32qk500-c-32-led-ips-quadhd-freesync
![image](https://user-images.githubusercontent.com/194400/100292668-05d00800-2f79-11eb-8a23-644008b4ce16.png)

**Comment by nelsonic** [comment]
LG 32UN880-B is not a "budget" option, it's an "UltraFine".
https://www.lg.com/us/monitors/lg-32un880-b-4k-uhd-led-monitor
![image](https://user-images.githubusercontent.com/194400/111887606-054af980-89ce-11eb-8b61-c7da7a80f242.png)

Watch this brief review: https://youtu.be/fKj9Sz9vCaQ?t=55
[![image](https://user-images.githubusercontent.com/194400/111887996-a20e9680-89d0-11eb-8351-231817c93910.png)](https://youtu.be/fKj9Sz9vCaQ?t=55)

The stand is very sturdy and versatile:
![image](https://user-images.githubusercontent.com/194400/111888029-eb5ee600-89d0-11eb-818c-a505ed49f09d.png)


I'm not bothered by the off-center stand. but apparently the panel inside is _exceptional_. 
Cheapest I've found in PT is PCComponentes:
https://www.pccomponentes.pt/lg-32un880-b-315-led-ips-ultrahd-4k-freesync-usb-c
![image](https://user-images.githubusercontent.com/194400/111887551-98376400-89cd-11eb-8d65-1f3f9a3a0e7a.png)

**Comment by iteles** [comment]
You have a refund for an order that never arrived, awaiting a response (of which account to return the money to) with PCComponentes so that’s handy.

**Comment by nelsonic** [comment]
@iteles yeah, PCComponentes saga ... 🤦‍♂️  (I sent them the IBAN for the refund... ⏳ )

Meanwhile Amazon ES has a _sizeable_ promo on the **38" Ultrawide** researched above https://github.com/nelsonic/poang-hack/issues/2#issuecomment-732514525
https://www.amazon.es/-/pt/dp/B0798RDFK7
![image](https://user-images.githubusercontent.com/194400/129636472-1adcdd57-273d-40df-a269-1d6215980749.png)

This feels almost too good to be true, but it's the LG Store on Amazon and fulfilled by Amazon. 💭

**Comment by nelsonic** [comment]
@iteles didn't get a reply from you on the VAT question (I sent to you via Signal) and now the price is back up to "normal" ... 😢 

![image](https://user-images.githubusercontent.com/194400/130109207-81ba1376-8ed4-4148-a7f7-79e54738831e.png)

Meanwhile, got the refund from PC Componentes:
![image](https://user-images.githubusercontent.com/194400/130109348-9547aec5-307e-4c83-af0d-fefd458ef271.png)

**Comment by nelsonic** [comment]
Would have _loved_ to get the 38" Ultra-wide (_especially at this discount_) 💰 
for video editing as it has _way_ more space for the timeline ... 😮 
But realistically I haven't done that much video editing this year ... 😢 
_Really_ looking forward to the end of my current contract (October 2021) so I can get back to my _plan_! ⏳ 
So I will re-visit this in November. Luckily "Black Friday" is November 26 and there should be a few bargains. 💭

**Comment by iteles** [comment]
This sucks. But I emailed Carlos as we discussed via signal to ask about the Iva number and he hasn’t replied yet. He told us most of his staff were on holiday this week so I suspect that’s why 🤦🏻‍♀️
Also didn’t realize it was such a limited time thing 😭

**Comment by nelsonic** [comment]
@iteles all price “anomalies” are time-sensitive. ⏳
Thanks for following up with Carlos. 👍🏻

**Comment by nelsonic** [comment]
My Apple Thunderbolt Display which I use every day for "work work" has developed the ["buzz of death"](https://blog.netnerds.net/2017/01/a-suggested-fix-for-thunderbolt-display-buzzing/#:~:text=The%20one%20that%20seemed%20to,crank%20up%20the%20display%20brightness.) (_the power-supply overheats and makes an unbearable buzzing noise! the only way to calm it is to switch off and **un-plug** the monitor and "take a break from screens" ... obviously not great if I have work to do ..._)

I've looked up fixing the power supply unit on the display (_can't find the thunderbolt display on any of the repair services because it's discontinued ... but an analog would be the 27" iMac which shares the same panel ..._), 
Changing the power supply is **`289,95 €`** ...
https://iservices.pt/reparacao/apple/mac/imac-27/problema/supply-power
![image](https://user-images.githubusercontent.com/194400/131208878-d03c29ca-1856-4188-aa8b-7a7ece0d0143.png)

The cost of the power supply unit on eBay is $160 and then I would still need to spend 2h figuring out how to change it ...
https://www.ebay.com/itm/332556630940
![image](https://user-images.githubusercontent.com/194400/131209038-6f3d4e83-713e-4531-9bba-e736639bd9d9.png)



These two Samsung monitors aren't ultra-wide so they wouldn't be for video editing,
but given that my "main" (_work_) monitor is now buzzing to the point of being unusable,
I'm seriously considering an upgrade ...

https://www.pcdiga.com/monitor-samsung-j590-va-31-5-4k-uhd-16-9-60hz-freesync-lu32j590uquxen
![image](https://user-images.githubusercontent.com/194400/131191859-51f27a33-702b-497e-8f5b-3a4554eaebd9.png)

More expensive, but better reviews: https://www.pccomponentes.pt/samsung-ls32a700nwuxen-32-led-ultrahd-4k
![image](https://user-images.githubusercontent.com/194400/131192468-948a4c1e-6687-4db6-9ecf-a2698fc60044.png)

**Comment by nelsonic** [comment]
The buzzing of my "main" Mac Monitor is extremely annoying. 
I've been getting around it by taking regular breaks and switching off (+ unplugging) it.
But it overheats _very_ quickly while using Microsoft Teams ( which I'm forced to use for "work work" ...🤮 )

This Samsung monitor is decent, has good reviews and is currently 23% discounted:
https://www.pccomponentes.com/samsung-lu32j590uqrxen-315-led-ultrahd-4k-freesync
![image](https://user-images.githubusercontent.com/194400/132326738-c0d3b837-d146-4c5c-92b7-00079f4ca5e9.png)

Spec:
![image](https://user-images.githubusercontent.com/194400/132327059-618f972f-4330-4e04-b0c8-4fca5c986fd8.png)

Order placed:
![image](https://user-images.githubusercontent.com/194400/132326634-03eac1f1-14a1-4f6a-8352-3c3c902aa116.png)

Should arrive in the next couple of days.

**Comment by nelsonic** [comment]
Monitor arrived. I set it up on my desk mount and it's amaze. Such incredible value-for-money. 

I'm not using the full 4K resolution:
![image](https://user-images.githubusercontent.com/194400/132776827-a5a83291-3676-4421-b102-cc45fc3a2f3e.png)

I can easily see 4 panels (90 cols wide and 80 rows) of code on this screen and there are no perceptible dark/light spots. 
We truly are living in the future. All this for £212.75 (sterling). 🤯

**Comment by nelsonic** [comment]
Been using the new monitor for the past 10 days and it's been a game-changer when writing code. No regrets.

**Comment by nelsonic** [comment]
If I were on the market for a **work** (code) monitor, I would _seriously_ consider the **HUAWEI MateView**

https://consumer.huawei.com/uk/monitors/mateview/specs/
![image](https://user-images.githubusercontent.com/194400/168766173-55d4d5c5-ef6a-42fc-b4d3-f6a032e83014.png)

Because of the **3:2** aspect ratio, it has _considerably_ more **_vertical_ pixels** than most other monitors:
![image](https://user-images.githubusercontent.com/194400/168768692-f418b942-8e6d-42e3-bb6b-ec691bf7199a.png)

The Monitor I've Always Wanted! (4K | 3:2 | 28"): https://youtu.be/DPttbRBOgdA
[![image](https://user-images.githubusercontent.com/194400/168764530-c0f2dfbf-0bc8-470d-bfe8-838f8bc4e905.png)](https://youtu.be/DPttbRBOgdA)

Currently £499 in the UK: https://consumer.huawei.com/uk/monitors/mateview/buy/
![image](https://user-images.githubusercontent.com/194400/168765164-07823cd5-640f-436c-886e-fe193c3c3359.png)

It's a third of the price of the Apple Studio Display and is bigger and has more vertical pixels.
https://www.apple.com/uk/shop/buy-mac/apple-studio-display
![image](https://user-images.githubusercontent.com/194400/168765746-6d8bf439-0385-429d-a89c-23027ea7a9e8.png)

Even though the specs on the Apple display are great:
![image](https://user-images.githubusercontent.com/194400/168771595-8da1e50a-f614-4fb0-a590-bced32da4adf.png)

And it has a built-in web cam ... but I find that my _external_ logitech camera serves my needs.

**Comment by nelsonic** [comment]
This is my *current* desk setup: 
![ED579522-D85B-4E57-AC95-A9B7E06DFE68](https://user-images.githubusercontent.com/194400/170079559-3446d1ed-9f20-48b6-bad6-e1cd26c5a04a.jpeg)

Not the best, but works well for me. 👌🏻

**Comment by nelsonic** [comment]
This looks siiiiick: https://www.lg.com/us/monitors/lg-40wp95c-w via: @hyprstack DM.
![image](https://user-images.githubusercontent.com/194400/170471265-41b6d5f8-c0d6-44c9-90ff-a95bdec8c32c.png)

![image](https://user-images.githubusercontent.com/194400/170471428-dd109ffb-ae7c-486d-99d0-bd7da453e09e.png)

Lg 40WP95C-W 40 in 5K UltraWide Monitor Review: https://youtu.be/Lwr7zC2mpys
[![image](https://user-images.githubusercontent.com/194400/170471796-a423a89c-0064-44d6-9e7d-ae5c43ce38a8.png)](https://youtu.be/Lwr7zC2mpys)

If I had one of the fancy new MacBook laptops (with USB-C) this would be a serious consideration as it has Power Delivery ... the single-cable dream. 😉 

Can't find a UK price/supplier ... see:
https://www.reddit.com/r/ultrawidemasterrace/comments/t6nwp7/lg_40wp95c_available_in_uk_this_month/

**Comment by hyprstack** [comment]
@nelsonic Have also watched this review. Seems like the best option available right now. It's a shame how Mac's scale though. Retina requires you PPI be in the 220 ballpark or non Retina be in the 110 PPI, which is essentially a 1440 resolution. But I'm willing to give it a go anyway. 40" is a lot of real-estate for work.

As for a supplier, I have found only one so far in the UK: [avpartsmaster](https://www.avpartsmaster.co.uk/lg-40wp95c-w-39-7-curved-ultrawide-5k2k-nano-ips-display-p176668/s176681?utm_source=TAG&utm_medium=affiliate&utm_campaign=%7BTAG.AID%7D&tagrid=%7BTAG.RID%7D&gclid=CjwKCAjw7cGUBhA9EiwArBAvokzLTfvUEHSbYCrTZkk4PbNQB99DoLI02Hdowdf5-z5NFfF_vxpI0BoCl08QAvD_BwE) 
Sadly never heard of them before 🤔

**Comment by nelsonic** [comment]
@hyprstack Oh wow **`£1,678.80`** ... 💸  😜 

https://www.avpartsmaster.co.uk/lg-40wp95c-w-39-7-curved-ultrawide-5k2k-nano-ips-display-p176668/s176681
![image](https://user-images.githubusercontent.com/194400/170685312-5e6b91d1-215d-44a5-8c2b-23bddacda06c.png)

Lucky you get the 20% VAT back now so not as much of a hit ... 👌 
Supplier seems legit tho. 

But ... at that price, you're in Apple Studio Display territory https://www.apple.com/uk/studio-display/

![image](https://user-images.githubusercontent.com/194400/170696067-0d8cc7cd-b6e2-4490-ae9d-f510286f6b63.png)


The comparison of resolution is: **`5120x2160`** vs **`5120x2880`** i.e. _much_ finer on the Apple Display.
And if you plan to use MacBook Pro 14 (or 16) as your **`main`** computer for the foreseeable future,
the **UX** might be _that_ much better. The higher sharpness, built-in 12Mp Camera, Superior Microphones, Speakers and brightness might be worth considering ... I know **27"** might feel "small" compared to **40"** ... but the ***content*** won't be "***stretched***" on the Apple Display. Whereas it will always feel stretched on the Ultrawide. 
The Studio Display is twice as Bright as the LG Ultrawide  **`600 nits`** vs **`300 nits`** which will make a difference to eye-strain after a few hours of use. 

> The other thing that I'v realised about buying Apple products is that they hold their value remarkably well.

But then Marques calls it an "oddball":
[![image](https://user-images.githubusercontent.com/194400/170692532-5cb37b71-74bd-4059-a8f4-5325c94455ef.png)](https://youtu.be/yoigsHYc77s)
Worth watching the review if you're on-the-fence ... the build quality looks sick.

# tl;dr

Yeah, get the ultra-wide. 😉 
If you don't _love_ it, you can easily sell it on. 💭 
Keep the box. 📦

**Comment by nelsonic** [comment]
YEYIAN ODRAZ **`32”`** PC **Gaming _Frameless_ `4K`** UHD HDR Monitor 3840x2160 **`IPS`** Panel, 1 Billion Colors, Low Blue, 350cd/m2, 1000:1, 60Hz, 4ms, 16:9, 178°, G-Sync, FreeSync, DP/HDMI/USB, Speakers, VESA, Tilt
https://www.amazon.com/YEYIAN-Frameless-Monitor-3840x2160-Billion/dp/B09HN7PBPL?th=1
<img width="1599" alt="image" src="https://user-images.githubusercontent.com/194400/201665405-f2ca1230-9238-45ad-aa26-2f95bd1deabc.png">

The name is a bit of a mouthful. Clearly an attempt to keyword stuff the Title for SEO. 
But the reviews seem decent. 

**`Yeyian`**? Yeah, never heard of them. 🤷‍♂️ 
They appear to be gaming-focussed ... 🎮 

https://www.amazon.com/stores/YeyianUSA/page/44AF436C-4519-4161-9076-F00A0A01F37A?ref_=ast_bln
<img width="1328" alt="image" src="https://user-images.githubusercontent.com/194400/201667518-e68bc41b-141c-4a85-b85a-85d263bbd3e9.png">

This Gamin monitor looks decent: as in high refresh rate and low response time:
https://www.amazon.com/YEYIAN-SIGURD-Frameless-Multistand-Monitor/dp/B09F4TV95F?ref_=ast_sto_dp&th=1
<img width="1662" alt="image" src="https://user-images.githubusercontent.com/194400/201667882-48655067-dfa3-4617-b701-ec80bf3e9802.png">

**`€270`** from PCDiga: 
https://www.pcdiga.com/monitor-yeyian-odraz-ips-32-4k-uhd-16-9-60hz-freesync-ymg-4k32-01-7503035006489
<img width="1524" alt="image" src="https://user-images.githubusercontent.com/194400/201666389-d2e9e3ed-6a33-4d02-819a-26addd7dae4d.png">

**Comment by nelsonic** [comment]
https://www.pccomponentes.pt/samsung-ls32bm700uuxen-smart-monitor-m7-32-led-ultrahd-4k-usb-c
<img width="1685" alt="image" src="https://user-images.githubusercontent.com/194400/201669246-7ed81cf1-72f1-4512-8e8b-6d92c7640933.png">

**Comment by nelsonic** [comment]
Samsung Smart Monitor M7 Review 2021: https://youtu.be/remYeLICJG0
<img width="1112" alt="image" src="https://user-images.githubusercontent.com/194400/201670002-0d6ab012-74bd-4dd0-84d3-8f23faa8f44a.png">

Buuuuut ... the USB-C PD only does 30hz at 4K res ...
https://us.community.samsung.com/t5/Monitors-and-Memory/M7-Smart-Monitor-32m70a-USB-c-issue/td-p/1642563

**Comment by nelsonic** [comment]
So ... I'm up-selling myself on getting a high color accuracy monitor for video editing ... 💭 💸 🤦‍♂️ 

https://www.lg.com/us/monitors/lg-32UL950-W-4k-uhd-led-monitor
![image](https://user-images.githubusercontent.com/194400/206359050-77f623fa-d0ae-47f3-9fff-508f75fbe8b9.png)


https://www.senetic.pt/product/32UL950-W
![image](https://user-images.githubusercontent.com/194400/206357974-cf5cb712-e114-47f3-af36-92781fbb28ca.png)

**Comment by nelsonic** [comment]
https://www.pccomponentes.pt/lg-ultrafine-ergo-32un88a-w-315-led-ips-ultrahd-4k-freesync-usb-c
![image](https://user-images.githubusercontent.com/194400/206360760-40288a08-7fb6-4eb5-b89b-9a98da86736c.png)

**Comment by nelsonic** [comment]
I like this one because of the integrated stand, cable management and USB-C power delivery (65W) 

https://www.lg.com/uk/monitors/lg-32un88a-w
![image](https://user-images.githubusercontent.com/194400/206361045-e5def275-15f1-422f-b250-c3e7b6af2b00.png)

![image](https://user-images.githubusercontent.com/194400/206361077-2dc091b4-cc58-4a2a-9e1f-4304c3e344ab.png)

**Comment by nelsonic** [comment]
https://www.pccomponentes.pt/usuarios/panel/mis-pedidos-y-facturas
![image](https://user-images.githubusercontent.com/194400/206363936-cc0418a5-3cda-4861-b7f9-af26cbe1ffff.png)

**Comment by nelsonic** [comment]
Arrived much faster than expected: 🎉

![D88F75F5-DC48-484C-B2F2-32F17BFEC5F3](https://user-images.githubusercontent.com/194400/206729910-e0130ef0-6b80-4e9b-99fc-9a759e2a930c.jpeg)

not going to unbox during the work day … only have 2h left and *loooots* todo!! ⏳

**Comment by hyprstack** [comment]
@nelsonic it also says on the middle right edge of the box "Do not open here". I guess you're gonna have to wait until that message changes 😃

**Comment by nelsonic** [comment]
LoL! I'm _itching_ to open it. but have to tidy my desk/office before I do any of that. ⏳ 
Hope your ***Friday*** is going well. 😜

**Comment by nelsonic** [comment]
Couple of weeks later the monitor costs €150 more ...
https://www.kuantokusta.pt/p/196943/lg-32-ergo-32un880-b-ips-uhd-4k-usb-c
<img width="1029" alt="image" src="https://user-images.githubusercontent.com/194400/216106696-df32dd75-7bbd-472c-926e-99eae36bd16b.png">

Been using it for the past few weeks; _Zero_ regrets!

**Comment by nelsonic** [comment]
Tempted to get a second one: https://www.ebay.co.uk/itm/134622854861
<img width="1560" alt="image" src="https://github.com/nelsonic/poang-hack/assets/194400/bbe1d9d7-d183-4178-bc4b-a9f362dd08a2">

Not for my desk but the "hot" desk in the library so _someone_ can work there when needed. 💭

**Comment by iteles** [comment]
I'm honestly not keen on the hot desk in the library so I would hold off until we've had a chance to discuss that 👌🏻

**Comment by nelsonic** [comment]
@iteles perhaps I didn't make myself clear. The Hot Desk that is _currently_ in the library but will _definitely_ move somewhere `else` once the space is reconfigured. 👌

**Comment by nelsonic** [comment]
Current Amazon Price for this monitor is: **`€547,90`**:
https://www.amazon.es/-/pt/dp/B0CR52JZVJ/
<img width="2032" alt="Screenshot 2024-09-27 at 10 32 05" src="https://github.com/user-attachments/assets/57e35e1a-eb56-48c7-9c0e-61b95b86d442">

But there are a couple available refurbished cheaper: 
<img width="585" alt="image" src="https://github.com/user-attachments/assets/1e66b81e-570b-4860-a380-67e75915233d">


> **Edit**: context: we have a new person arriving [**`@home`**](https://github.com/dwyl/home)
and it would cool to set them up with a decent second-screen monitor. 🖥️

**Comment by nelsonic** [comment]
Sadly, the USB-C feature of the LG Ultrafine has stopped cooperating. No signal. No charging. 😢 
I'm not the only one with this problem:
https://mactrovert.com/lg-32un880-b-display-issues-with-m1-mac-mini-when-using-usb-c/

**Comment by nelsonic** [comment]
Price dropped on "like new" (second hand) monitor so I picked up another one:
<img width="589" alt="image" src="https://github.com/user-attachments/assets/0185ed9a-d819-4187-aac4-fe4d06d360e0">

We have _two_ apprentices in the house right now so they will each have an _awesome_ monitor to work with. 🖥️ 😍

**Comment by nelsonic** [comment]
HP OMEN 32q: https://www.amazon.es/dp/B0C4F38PXP trades-off resolution for high refresh rate but is a _bargain_!

<img width="1628" alt="Screenshot 2024-10-08 at 12 41 25" src="https://github.com/user-attachments/assets/b778f3da-7a1f-4cb9-b714-944f66346867">

This is a _really_ good deal/price for a **32"** monitor. Tempted ... 😬

**Comment by nelsonic** [comment]
This Monitor is a _shameless_ clone of the **`Apple Pro Display XDR`**:
https://pt.aliexpress.com/item/1005006913151433.html
<img width="1528" alt="image" src="https://github.com/user-attachments/assets/4cfd7b54-d845-40dc-a02c-fe17ef8b34f4">

![1](https://github.com/user-attachments/assets/8bf2fbe7-899c-47da-8156-8163791edff7)
![2](https://github.com/user-attachments/assets/3b156690-1506-4a9c-96f5-76a9bed3c5c9)
![3](https://github.com/user-attachments/assets/238853fb-a34b-4dbf-941e-1c1626745d19)

_Obviously_ the panel on the _inside_ is not the same! It's just the _exterior_ that is cloned.

For reference, just the _stand_ on the **`Pro Display XDR`** is **`$1k`** ... 🤦‍♂️ 
https://www.apple.com/uk/shop/buy-mac/pro-display-xdr/standard-glass
<img width="1151" alt="image" src="https://github.com/user-attachments/assets/b5afc26b-e26d-4908-87ca-e919945fca83">


Speaking of thousand dollar monitor stands, those are available cloned too: 
https://www.aliexpress.com/item/1005007429674322.html
<img width="1543" alt="image" src="https://github.com/user-attachments/assets/a313d280-eb0e-4d82-b922-e8d31e97861a">

Obviously I'm not buying one of these. But if I did have a "spare" couple of grand sitting around ... 
I'd much sooner buy a couple of these clones than sink **`$5k`** into an Apple one. 💭
